src/csharp/Microsoft.Spark/Utils/UdfSerDe.cs

```diff
@@ -158,15 +227,15 @@ private static TargetData SerializeTarget(object target)
         private static object DeserializeTargetData(TargetData targetData)
         {
             Type targetType = DeserializeType(targetData.TypeData);
-            var target = Activator.CreateInstance(targetType);
+            var target = FormatterServices.GetUninitializedObject(targetType);
```

Comment:
`Activator.CreateInstance(targetType)'` requires targetType to have a ctor().  `FormatterServices.GetUninitializedObject(targetType);` allows the creation of a new instance of targetType without having to call the ctor().